student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super
```

Comment:
- [ ] This Class should inherit all the parent constructor parameters, so the Student constructor should have `age`, `classroom`, `name`, and `parent_permission` parameters. also super should contain those parameters. kindly make sure to implement this so the Student Class extends parent contracture